lib/beaker/subcommands/subcommand_util.rb

```diff
@@ -110,6 +112,65 @@ def self.init_hypervisor(options)
           init_vmpooler
         end
       end
+
+      def self.store
+        @@store
+      end
+
+      # Load the beaker config
+      def self.load_config()
+        FileUtils.mkdir_p ".beaker"
+        @@store = YAML::Store.new(".beaker/config")
+      end
+
+      # Store values from a hash into the beaker config
+      # @param [Array<Object>] config values we want to store
+      def self.store_config(config)
+        store.transaction do
+          store[:hypervisor] = config[:hypervisor] unless config[:hypervisor].nil?
+          store[:provisioned] = config[:provisioned] unless config[:provisioned].nil?
+          store[:hosts] = config[:hosts] unless config[:hosts].nil?
```

Comment:
So here, I'm explicitly stating what keys I want to write to. This means that we can pass a hash full of info to this method, but we only write out the info the subcommands need to know about